### PR TITLE
Correcting errors in API calls to brainspell

### PR DESCRIPTION
**Pull request by Sharabesh**
Correcting the problems associated with `edit-local-article`, `edit-global-article` and my `forcePullCollection` button from before. 

There were a couple of small issues on the frontend with how we were sending data back that could cause our article data to be corrupted in the database so I went ahead and made those modifications here. 

The corresponding backend fixes are here:
https://github.com/OpenNeuroLab/brainspell-neo/pull/71

### CODE REVIEW

File: brainspell/src/App.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
     -->
     <b-navbar toggleable="md" type="dark" variant="dark" sticky>
 
-      <b-navbar-toggle target="nav_collapse"></b-navbar-toggle>
+      <b-navbar-toggle target="nav_collapse"></b-navbar-toggle>`
```
Comment (62442katieb) [inline brainspell/src/App.vue]: is this ` intentional?

Comment (Sharabesh) [reply brainspell/src/App.vue]: nope sorry, corrected!

**Comment by Sharabesh** [comment]
Bumping did this look ok to merge?

**Comment by 62442katieb** [comment]
I think it looks good. I'll go ahead and merge.

**Pull request merged**